config/devel.yaml

```diff
@@ -6,7 +6,7 @@ PLAINTEXT_VARIABLES:
     SIERRA_DB_NAME: iii
     ENVISIONWARE_DB_PORT: 3306
     ENVISIONWARE_DB_NAME: lasttwodays
-    REDSHIFT_DB_NAME: qa
+    REDSHIFT_DB_NAME: dev
```

Comment:
yeah this is kinda a matter of preference -- a lot of the alarms won't actually work in devel because the tables don't exist, but maybe it's better to have the redshift db be dev anyway